lldb/tools/debugserver/source/debugserver.cpp

```diff
@@ -169,6 +170,7 @@ static bool is_dot_app (const char *app_name) {
     return true;
   return false;
 }
+#endif
```

Comment:
This function is also kind of sad, but debugserver does not link against LLVM and std::string::ends_with is C++20 only. Oh well.

Reply:
sadbatman.jpg